layout/password.liquid

```diff
@@ -37,6 +37,46 @@
       {{ body_font_bold_italic | font_face: font_display: 'swap' }}
       {{ settings.type_header_font | font_face: font_display: 'swap' }}
 
+      {% for scheme in settings.color_schemes %}
+        {% if forloop.index == 1 %}
+          :root,
+        {% endif %}
+        .color-{{ scheme.id }} {
+          --color-scheme-background: {{ scheme.settings.background.red }},{{ scheme.settings.background.green }},{{ scheme.settings.background.blue }};
+          {% if scheme.settings.background_gradient != empty %}
+            --color-scheme-background-gradient: {{ scheme.settings.background_gradient }};
+          {% else %}
+            --color-scheme-background-gradient: {{ scheme.settings.background }};
+          {% endif %}
+          --color-scheme-text: {{ scheme.settings.text.red }},{{ scheme.settings.text.green }},{{ scheme.settings.text.blue }};
+          --color-scheme-button: {{ scheme.settings.button.red }},{{ scheme.settings.button.green }},{{ scheme.settings.button.blue }};
+          --color-scheme-button-label: {{ scheme.settings.button_label.red }},{{ scheme.settings.button_label.green }},{{ scheme.settings.button_label.blue }};
+          --color-scheme-secondary-button-label: {{ scheme.settings.secondary_button_label.red }},{{ scheme.settings.secondary_button_label.green }},{{ scheme.settings.secondary_button_label.blue }};
+          --color-scheme-shadow: {{ scheme.settings.shadow.red }},{{ scheme.settings.shadow.green }},{{ scheme.settings.shadow.blue }};
+        }
+      {% endfor %}
+
+      {% for scheme in settings.color_schemes %}
+        .color-{{ scheme.id }},
```

Comment:
Is there a way to generate this style in the first loop?  (line 40)
Maybe we can set this in liquid variable?